README.md

```diff
@@ -57,26 +57,16 @@ To get the source code from GitHub, type
     cd ./azure-sdk-for-php
 
 > **Note**
-> 
+>
 > The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [Composer package manager](http://getcomposer.org).
 
-
 ##Install via Composer
+1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
+It can be installed locally or globally depending on your own needs.
 
-* Create a file named **composer.json** in the root of your project and add the following code to it:
-```json
-    {
-        "require": {        
-            "microsoft/windowsazure": "^0.4"
-        }  
-    }
-```
-
-* Download **[composer.phar](http://getcomposer.org/composer.phar)** in your project root.
-
-* Open a command prompt and execute this in your project root
+2. Execute the following command in order to get this package:
 
-    php composer.phar install
+    php composer create-project microsoft/windowsazure /path/to/install/dir ^0.4
```

Comment:
Did you try your command? php composer.phar create-project microsoft/windowsazure ./test2 ^0.4 will try to install v0.4.0, not v0.4.2.

php composer.phar create-project microsoft/windowsazure ./test3 ^0.4.2 does not work either